README.md

```diff
@@ -96,10 +125,44 @@ On default the chloroExtractor uses this config file, you can edit these one, or
 $ ./ptx -c ownptx.cfg -1 FQ_1 -2 FQ_2
 
 ```
+## Input data
+The chloroExtractor uses unsortet Fastq files with paired end reads. Please make sure your reads are not sortet at all, otherwise there could be problems or even wrong results.
+
+## Example
+An example data set can be downloaded from zenodo. As example we download the dataset into a folder and run chloroExtractor with the input files.
 
-## Data
-The chloroExtractor uses unsortet Fastq files with paired end reads. Please make sure your reads are not sortet at all, otherwise there could be problems or even wrong results. 
+For preparation, a folder will be created and a example dataset will be downloaded:
+
+```shell
+# create a folder for the testrun
+mkdir -p /tmp/chloroExtractor-testrun
+cd /tmp/chloroExtractor-testrun
+
+# download the example set and extract the sequencing reads
+wget 'https://zenodo.org/record/884449/files/SRR5216995_1M.tar.bz2' -O - | tar xjf -
+```
+Afterwards, chloroExtractor can be run in command line mode:
+
+```shell
+# run chloroExtractor via command line (assuming all dependencies are installed and ptx folder is in PATH)
+ptx -1 SRR5216995_1M_1.fastq -2 SRR5216995_1M_2.fastq
+[17-09-21 13:42:42] [PipeWrap] Running ptx from the beginning, no previous runs detected.
+[17-09-21 13:42:42] [PipeWrap] Running 'jf0': jellyfish count -t 8 -m 31 -s 500M -C -o jf0.jf /data/SRR5216995_1M_1.fastq /data/SRR5216995_1M_2.fastq
+[...]
+```
+
+or using the docker container:
+
+```shell
+# other possibility is docker container based chloroExtractor (assuming that the user is allowed to run docker)
+docker pull chloroextractorteam/chloroextractor # ensure the latest version from docker hub
+docker run -v /tmp/chloroExtractor-testrun:/data --rm chloroextractorteam/chloroextractor -1 SRR5216995_1M_1.fastq -2 SRR5216995_1M_2.fastq
+[17-09-21 13:52:30] [PipeWrap] Running ptx from the beginning, no previous runs detected.
+[17-09-21 13:52:30] [PipeWrap] Running 'jf0': jellyfish count -t 8 -m 31 -s 500M -C -o jf0.jf /data/SRR5216995_1M_1.fastq /data/SRR5216995_1M_2.fastq
+[...]
+```
 
+Both runs result in a final chloroplast assembly in the file `fcg.fa`
```

Comment:
Maybe a little more information about the other output files would be useful as well. We can also integrate some of the additional info I put into https://github.com/chloroExtractorTeam/chloroExtractor/blob/add-demo/DEMO.md

Reply:
Added a link to your DEMO.md and included the file via pull request into that branch. Maybe information about other output/log files should be included into the DEMO.md